.github/CODEOWNERS

```diff
@@ -938,6 +938,7 @@ x-pack/platform/plugins/shared/streams_app @elastic/streams-program-team
 x-pack/platform/plugins/shared/task_manager @elastic/response-ops
 x-pack/platform/plugins/shared/timelines @elastic/security-threat-hunting-investigations
 x-pack/platform/plugins/shared/triggers_actions_ui @elastic/response-ops
+x-pack/platform/test
```

Comment:
Should every entry has an associated team? I guess this line can be deleted

Reply:
I think it is automatically added by CI bot, I will explicitly list the owners for what's inside of it and has no owner yet.

Reply:
I found entries for:

```
x-pack/test
x-pack/test_serverless
```

so I guess it is expected to be added?